test fails while building debian package

```
Making target test for debian/lua5.1.dh-lua.conf
# tests
Copying luaunit.lua in /build/lua-unit-3.3/5.1-unit for test
****************** lua dynamic custom (5.1) ******
........................................................................................F..................................................................................................
Failed tests:
-------------
1) TestLuaUnitUtilities.test_prettystrTableRecursion
./test/test_luaunit.lua:496: Could not match pattern 
'(<table: 0?x?[%x]+>) {"t9", (<table: 0?x?[%x]+>) {}, (<table: 0?x?[%x]+>) {%1}=1}'
 with string 
'<table: 0x55f24343f780> {
    "t9",
    <table: 0x55f24343f7d0> {},
    <table: 0x55f24347e420> {<table: 0x55f24343f780>}=1
}'
stack traceback:
	./test/test_luaunit.lua:496: in function 'TestLuaUnitUtilities.test_prettystrTableRecursion'

Ran 187 tests in 0.015 seconds, 186 successes, 1 failure
make[2]: *** [/usr/share/dh-lua/make/dh-lua.Makefile.single:312: test-lua-dynamic-real-custom] Error 1
make[1]: *** [/usr/share/dh-lua/make/dh-lua.Makefile.multiple:12: test] Error 1
dh_auto_test: make --no-print-directory -f /usr/share/dh-lua/make/dh-lua.Makefile.multiple test returned exit code 2
make: *** [debian/rules:4: build] Error 2
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

--- a/test/test_luaunit.lua
+++ b/test/test_luaunit.lua
@@ -493,19 +493,7 @@ bar"=1}]] )
         local t9 = {"t9", {}}
         t9[{t9}] = 1
 
-        if os.getenv('TRAVIS_OS_NAME') == 'osx' then
-            -- on os X, because table references are longer, the table is expanded on multiple lines.
-            --[[ Output example:
-            '<table: 0x7f984a50d200> {
-                "t9",
-                <table: 0x7f984a50d390> {},
-                <table: 0x7f984a50d410> {<table: 0x7f984a50d200>}=1
-            }'
-            ]]
-            lu.assertStrMatches(lu.prettystr(t9, true), '(<table: 0?x?[%x]+>) {\n%s+"t9",\n%s+(<table: 0?x?[%x]+>) {},\n%s+(<table: 0?x?[%x]+>) {%1}=1\n}')
-        else
-            lu.assertStrMatches(lu.prettystr(t9, true), '(<table: 0?x?[%x]+>) {"t9", (<table: 0?x?[%x]+>) {}, (<table: 0?x?[%x]+>) {%1}=1}')
-        end
+        lu.assertStrMatches(lu.prettystr(t9, true), '(<table: 0?x?[%x]+>) {\n?%s*"t9",\n?%s*(<table: 0?x?[%x]+>) {},\n?%s*(<table: 0?x?[%x]+>) {%1}=1\n?}')
     end
 
     function TestLuaUnitUtilities:test_prettystrPairs()

